artemis-server/src/main/java/org/apache/activemq/artemis/core/settings/impl/AddressSettings.java

```diff
@@ -549,7 +569,8 @@ public int getEncodeSize() {
          BufferHelper.sizeOfNullableBoolean(autoDeleteJmsQueues) +
          BufferHelper.sizeOfNullableBoolean(autoCreateJmsTopics) +
          BufferHelper.sizeOfNullableBoolean(autoDeleteJmsTopics) +
-         BufferHelper.sizeOfNullableInteger(managementBrowsePageSize);
+         BufferHelper.sizeOfNullableInteger(managementBrowsePageSize) +
+         BufferHelper.sizeOfNullableLong(maxSizeBytesRejectThreshold);
```

Comment:
I suspect this could break compatibility on the bindings journal? better to check.